src/main/java/com/example/document/service/DocumentService.java

```diff
@@ -53,37 +55,55 @@ public List<DocumentEntryDTO> uploadExcelDocument(MultipartFile file) throws Exc
+                .mapToObj(i -> DocumentWithEmployeeDTO.of(documentEntryDTOS.get(i), employeeDTOs.get(i))).collect(Collectors.toList());
     }
 
-    public List<DocumentEntryDTO> insertDocumentEntries(List<DocumentEntryDTO> documentEntryDTOS) {
-        List<DocumentEntry> documentEntries = documentEntryMapper.dtoToEntity(documentEntryDTOS);
-        documentRepository.saveAll(documentEntries);
-        return documentEntryMapper.entityToDto(documentEntries);
+    public List<DocumentWithEmployeeDTO> insertDocumentEntries(List<DocumentEntryDTO> documentEntryDTOS) {
+        List<DocumentEntry> savedDocumentEntries = documentRepository.saveAll(documentEntryMapper.dtoToEntity(documentEntryDTOS));
+        List<DocumentEntryDTO> savedDocumentEntryDTOs = documentEntryMapper.entityToDto(savedDocumentEntries);
+
+        return savedDocumentEntryDTOs.stream()
+                .map(documentEntryDTO -> DocumentWithEmployeeDTO.of(documentEntryDTO, employeeClient.getEmployeeById(documentEntryDTO.getEmployeeId())))
+                .collect(Collectors.toList());
     }
 
-    public List<DocumentEntryDTO> getAllDocuments() {
-        List<DocumentEntry> allDocumentEntries = documentRepository.findAll();
-        return documentEntryMapper.entityToDto(allDocumentEntries);
+    public List<DocumentWithEmployeeDTO> getAllDocuments() {
+        List<DocumentEntryDTO> allDocumentEntryDTOs = documentEntryMapper.entityToDto(documentRepository.findAll());
+
+        return allDocumentEntryDTOs.stream()
+                .map(documentEntryDTO -> DocumentWithEmployeeDTO.of(documentEntryDTO,employeeClient.getEmployeeById(documentEntryDTO.getEmployeeId()))).collect(Collectors.toList());
     }
 
     public DocumentWithEmployeeDTO viewDocumentEntry(Long id) {
         DocumentEntry documentEntry = documentRepository.findById(id).orElse(null);
         DocumentEntryDTO documentEntryDTO = documentEntryMapper.entityToDto(documentEntry);
         EmployeeDTO employeeById = employeeClient.getEmployeeById(documentEntryDTO.getEmployeeId());
-        employeeById.setEmployeeId(documentEntry.getEmployeeId());
         return DocumentWithEmployeeDTO.of(documentEntryDTO, employeeById);
     }
 
-    public List<DocumentEntryDTO> viewMultipleDocumentEntries(LocalDate startDate, LocalDate endDate) {
+    public List<DocumentWithEmployeeDTO> viewMultipleDocumentEntries(LocalDate startDate, LocalDate endDate) {
         List<DocumentEntry> documentEntry = documentRepository.findByEffectiveDateBetween(startDate, endDate);
-        return documentEntryMapper.entityToDto(documentEntry);
+        List<DocumentEntryDTO> documentEntryDTOS = documentEntryMapper.entityToDto(documentEntry);
+
+        List<EmployeeDTO> employeeDTOs = IntStream.range(0, documentEntryDTOS.size())
+                .mapToObj(i -> employeeClient.getEmployeeById(documentEntryDTOS.get(i).getEmployeeId()))
+                .collect(Collectors.toList());
+        return IntStream.range(0, documentEntryDTOS.size())
```

Comment:
Let's replace all streams in this class with newer ones
And also let's put `.collect...` to next line to `.map...`